server/src/librelay/relay_server_test.go

```diff
@@ -184,13 +184,13 @@ func NewRelay(relayHubAddress common.Address) {
0919050565b6000602082019050818103600083015261591181614e0c565b9050919050565b6000602082019050818103600083015261593181614e4c565b9050919050565b6000602082019050818103600083015261595181614e8c565b9050919050565b6000602082019050818103600083015261597181614ecc565b9050919050565b6000602082019050818103600083015261599181614f0c565b9050919050565b600060208201905081810360008301526159b181614f4c565b9050919050565b600060208201905081810360008301526159d181614fb2565b9050919050565b600060208201905081810360008301526159f181614ff2565b9050919050565b60006020820190508181036000830152615a1181615058565b9050919050565b60006040820190508181036000830152615a328185615098565b9050615a4160208301846151b9565b9392505050565b6000602082019050615a5d60008301846151b9565b92915050565b6000604082019050615a7860008301856151b9565b615a8560208301846151b9565b9392505050565b600060a082019050615aa160008301886151b9565b615aae60208301876151b9565b615abb60408301866151b9565b615ac86060830185614566565b615ad56080830184614719565b9695505050505050565b6000608082019050615af460008301876151b9565b615b0160208301866151b9565b615b0e60408301856151b9565b8181036060830152615b208184614728565b905095945050505050565b6000604051905081810181811067ffffffffffffffff82111715615b4e57600080fd5b8060405250919050565b600067ffffffffffffffff821115615b6f57600080fd5b601f19601f8301169050602081019050919050565b600067ffffffffffffffff821115615b9b57600080fd5b601f19601f8301169050602081019050919050565b600067ffffffffffffffff821115615bc757600080fd5b601f19601f8301169050602081019050919050565b600081519050919050565b600081519050919050565b600081519050919050565b600081519050919050565b600082825260208201905092915050565b600082825260208201905092915050565b600081905092915050565b600082825260208201905092915050565b600081905092915050565b6000615c5c82615cdd565b9050919050565b6000615c6e82615cdd565b9050919050565b60008115159050919050565b6000819050919050565b60007fffffffff0000000000000000000000000000000000000000000000000000000082169050919050565b6000819050615cc582615e06565b919050565b6000819050615cd882615e13565b919050565b600073ffffffffffffffffffffffffffffffffffffffff82169050919050565b6000819050919050565b600060ff82169050919050565b6000615d1f82615d4a565b9050919050565b6000615d3182615cb7565b9050919050565b6000615d4382615cca565b9050919050565b6000615d5582615d5c565b9050919050565b6000615d6782615cdd565b9050919050565b82818337600083830152505050565b60005b83811015615d9b578082015181840152602081019050615d80565b83811115615daa576000848401525b50505050565b6000615dbb82615dcc565b9050919050565b6000819050919050565b6000615dd782615df9565b9050919050565b6000819050919050565b6000601f19601f8301169050919050565b60008160601b9050919050565b60058110615e1057fe5b50565b60048110615e1d57fe5b50565b615e2981615c51565b8114615e3457600080fd5b50565b615e4081615c81565b8114615e4b57600080fd5b50565b615e5781615c8b565b8114615e6257600080fd5b50565b60058110615e7257600080fd5b50565b615e7e81615cfd565b8114615e8957600080fd5b5056fea365627a7a72305820140e1a345ff09aea1fd0f8d57c8980da2a20ebc054905960a535065b418bd3f36c6578706572696d656e74616cf564736f6c634300050a0040"
```

Comment:
As I wrote in another comment, this isn't necessary and can be fixed without changing the original Go code:



In order to solve it, you can have these changes:

    Change server/scripts/get_abi_bin.js:
    var contractsToExtract =[ "IRelayHub", "RelayHub", "SampleRecipient" ];
    Change server/Makefile:
    instead of TRUFFLE_OUT ( which should probably be renamed to RELAYHUB_OUT), have two vars:
    RELAYHUB_BIN=$(buildpath)/../contracts/RelayHub.json
    RELAYHUB_ABI=$(buildpath)/../contracts/IRelayHub.json

then use abigen in the GEN_FILE target like that:
abigen --solc ../node_modules/.bin/solcjs --abi $(subst contracts,server/contracts,$(RELAYHUB_ABI:json=abi)) --bin $(subst contracts,server/contracts,$(RELAYHUB_BIN:json=bin)) --pkg librelay --type RelayHub --out $@

then you wouldn't need to change any Go code.